core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -1,11 +1,8 @@
 {% extends 'pages/base.html' %}
 
 {% block maintitle %}
-  {% if title %}
-    {{ title }} - Oppia Editor
-  {% else %}
-    Untitled Exploration - Oppia Editor
-  {% endif %}
+<div ng-if="title">[title]-Oppia Editor</div>
```

Comment:
Indent this line by 2.
Same for line 5.

Reply:
done